code/modules/nano/interaction/default.dm

```diff
@@ -42,7 +42,7 @@ var/global/datum/topic_state/default/default_topic_state = new
 		return STATUS_CLOSE
 
 	// If an object is in view then we can interact with it
-	if(src_object in view(get_effective_view(client), src))
+	if(src_object in view(client.view, src))
```

Comment:
Again, not sure what the point of this is.

Reply:
When checking view() and using user view, the proc returns non-wide supported view (21x15 for example).